### DISCUSSION TITLE
[6주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#50] [General]
### 진행순서
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Network] Web Server와 WAS #47 
2. 이주형
 * [Database] Concurrency control(동시성 제어) #49  
3. 김정윤
 * [Network] WebSocket #43 
4. 이동호
 * [Network] TCP #46

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by KJY97** [reply]
- 느낀점
  네트워크 어렵네요. 공부 열심히 해야겠어요..ㅠㅠ

**Reply by ChaerinYu** [reply]
* 좀 더 친절한 설명 준비하도록 하겠습니다.. . . 저의 역량.. 너무 부족하네요.. ..

**Reply by doho-ho** [reply]
~~@ChaerinYu 이 이모티콘으로 욕해요~~

- 흥미로운 주제가 많았습니다.
- ~~욕 좀 줄여주세요~~
- 이런거구나 하는 것과 정말 이해를 하고 정리해서 설명하는건 다르다는 것을 느꼈습니다.

**Reply by leeejuhyeong** [reply]
- 좋았던 점
오늘 내용 모두 유익했습니다
- 느낀점
저는... 저거 분할하고 뺄꺼 빼고 다시 준비해올게요
